api/config/ingest/config_request.go

```diff
@@ -26,6 +26,8 @@ func DefaultConfigRequest() *ConfigRequest {
 	c.V1.Sys.Service.Port = w.Int32(10122)
 	c.V1.Sys.Service.PurgeConvergeHistoryAfterDays = w.Int32(30)
 	c.V1.Sys.Service.PurgeActionsAfterDays = w.Int32(30)
+	c.V1.Sys.Service.MaxNumberOfBundledRunMsgs = w.Int32(10000)
```

Comment:
💭 I wonder if we want to lower this default for run messages when where.  At 10,000 messages, we'll hit the es-gateway limit at an expected message size of 25KB.  I think in practice this doesn't matter because most users don't see a request rate that can even get close to bundling that many messages.

Reply:
@stevendanna Do you have a guess at what the average message size is for our users?

Reply:
I'm not sure how representative this is, but at one customer, we recorded a few thousand API calls and saw the following percentiles in terms of data_collector client body sizes at the frontend lb:

```
     50%      90%      95%      99%    99.5%    99.9%
 71367.0 127934.0 140055.0 213987.0 237235.0 385558.2
```

I believe those are in bytes. So roughly 70k median request size.

(more info on where those numbers came from in the old history https://github.com/chef/a2/pull/4238)

Reply:
Does a default of 2,500 seem correct? That allows an average message size of 100KB.

Reply:
That seems reasonable to me.